doc/source/quickstart.rst

```diff
@@ -127,6 +121,13 @@ to compile CESM2 on a compute node as follows:
 
```

Comment:
A few lines up: delete the word "the" before qcmd. Also, a better link would be https://www2.cisl.ucar.edu/resources/computational-systems/cheyenne/running-jobs/submitting-jobs-pbs